boa/src/builtins/array/mod.rs

```diff
@@ -487,6 +488,45 @@ impl Array {
         Ok(a.into())
     }
 
+    ///'Array.prototype.at(index)'
+    ///
+    /// When at method takes desired integer as index and returns the value at given
+    /// index. Negative integer counts backwards. If index is invalid, the at method
+    /// returns undefined.
```

Comment:
Could you clarify this comment? You could instead take the explanation of the function on MDN and paste it here, that should be enough to explain what this does.

Reply:
Took the MDN description. I thought that one was the best too, but I was a bit hesitant to just grab it.